src/main/java/seedu/address/MainApp.java

```diff
@@ -81,12 +81,12 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
             addressBookOptional = storage.readAddressBook();
             if (!addressBookOptional.isPresent()) {
                 logger.info("Creating a new data file " + storage.getAddressBookFilePath()
-                        + " populated with a sample AddressBook.");
+                        + " populated with a sample ClinicMate.");
```

Comment:
maybe change it to 'a sample version of ClinicMate'? the one u have sounds a little weird.